README.md

```diff
@@ -22,7 +22,7 @@ provider "ibm" {
 
 module "redis" {
   # replace "main" with a GIT release version to lock into a specific release
-  source            = "terraform-ibm-modules/icd-redis/ibm"
+  source            = "terraform-ibm-modules/terraform-ibm-icd-redis/ibm?ref=main"
```

Comment:
Please change the source back, it was already correct. This is wrong

Reply:
its missing the version: `version = "X.X.X" # Replace "X.X.X" with a release version to lock into a specific release`